packages/circuit-ui/components/Selector/Selector.tsx

```diff
@@ -77,12 +77,14 @@ export interface SelectorProps
 type LabelElProps = Pick<SelectorProps, 'disabled' | 'size' | 'noMargin'>;
 
 const baseStyles = ({ theme }: StyleProps) => css`
-  display: inline-block;
+  display: flex;
```

Comment:
I believe this is a breaking change since it changes the `Selector` from an inline to a block-level element. Should this have been `inline-flex`?